roachtest: sqlalchemy failed

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/19013772?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/19013772?buildTab=artifacts#/sqlalchemy) on release-25.1 @ [9ae4c83eb6a018b1d46acdb30187ea07fdba5f5a](https://github.com/cockroachdb/cockroach/commits/9ae4c83eb6a018b1d46acdb30187ea07fdba5f5a):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.1.1-dev-9ae4c83eb6a018b1d46acdb30187ea07fdba5f5a
Tests run against sqlalchemy 2.0.23
542 Total Tests Run
540 tests passed
2 tests failed
716 tests skipped
1 test ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test/test_suite_sqlalchemy.py::IsolationLevelTest_cockroachdb+psycopg2_9_5_0::test_non_default_isolation_level (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::IsolationLevelTest_cockroachdb+psycopg2_9_5_0::test_all_levels (unexpected)
For a full summary look at the sqlalchemy artifacts 
An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
test artifacts and logs in: /artifacts/sqlalchemy/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-19013772/sqlalchemy/1741249535726/1741249760334)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-48307

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/19028435?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/19028435?buildTab=artifacts#/sqlalchemy) on release-25.1 @ [dff9f8466e36573024691a7f1be823071914d174](https://github.com/cockroachdb/cockroach/commits/dff9f8466e36573024691a7f1be823071914d174):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.1.1-dev-dff9f8466e36573024691a7f1be823071914d174
Tests run against sqlalchemy 2.0.23
542 Total Tests Run
540 tests passed
2 tests failed
716 tests skipped
1 test ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test/test_suite_sqlalchemy.py::IsolationLevelTest_cockroachdb+psycopg2_9_5_0::test_all_levels (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::IsolationLevelTest_cockroachdb+psycopg2_9_5_0::test_non_default_isolation_level (unexpected)
For a full summary look at the sqlalchemy artifacts 
An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
test artifacts and logs in: /artifacts/sqlalchemy/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-19028435/sqlalchemy/1741334721090/1741334955282)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #142446 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-master release-blocker]
- #142394 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-24.3.7-rc]
- #142393 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.1.1-rc]
- #142390 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-24.1.14-rc]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/cmd/roachtest/tests/sqlalchemy_blocklist.go
+++ b/pkg/cmd/roachtest/tests/sqlalchemy_blocklist.go
@@ -8,5 +8,7 @@ package tests
 var sqlAlchemyBlocklist = blocklist{}
 
 var sqlAlchemyIgnoreList = blocklist{
-	`test/test_suite_sqlalchemy.py::IsolationLevelTest_cockroachdb+psycopg2_9_5_0::test_default_isolation_level`: "roachtest configures the default isolation level different to what the text expects",
+	`test/test_suite_sqlalchemy.py::IsolationLevelTest_cockroachdb+psycopg2_9_5_0::test_all_levels`:                  "roachtest configures the default isolation level different to what the test expects",
+	`test/test_suite_sqlalchemy.py::IsolationLevelTest_cockroachdb+psycopg2_9_5_0::test_default_isolation_level`:     "roachtest configures the default isolation level different to what the test expects",
+	`test/test_suite_sqlalchemy.py::IsolationLevelTest_cockroachdb+psycopg2_9_5_0::test_non_default_isolation_level`: "roachtest configures the default isolation level different to what the test expects",
 }

